### DISCUSSION TITLE
Please help (Airdrop project code)

### DISCUSSION THREAD

**Discussion by codypharm** [Cyfrin/foundry-full-course-cu#2208] [Q&A] [answered]
I added zksync to my airdrop project and all of a sudden, tests that passed before started failing.
https://github.com/codypharm/airdrop

here is my code.

even removing the test and just loggin the user, I still get error

Failing tests:
Encountered 1 failing test in test/MerkleAirdrop.t.sol:MerkelAirdropTest
[FAIL. Reason: EvmError: Revert] testUserCanClaim() (gas: 255)

**Comment by codypharm** [comment, marked as answer]
Okay found out foundryup-zksync messes with the addresses or accounts so I did foundryup again to reset all and skipping zksync for now

**Reply by EngrPips** [reply]
Working with `ZkSync` on Foundry is not really smooth. I will say avoid it for now unless you really need to use it. I expect this to get better over time.

**Reply by codypharm** [reply]
Yea , thanks
